plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/jdbc/JDBCUtils.java

```diff
@@ -687,7 +687,7 @@ private static void debugColumnRead(ResultSet dbResult, int columnIndex, Excepti
         debugColumnRead(dbResult, "#" + columnIndex, error);
     }
 
-    public static void appendFilterClause(StringBuilder sql, DBSObjectFilter filter, String columnAlias, boolean firstClause)
+    public static void appendFilterClause(StringBuilder sql, DBSObjectFilter filter, String columnAlias, boolean firstClause, SQLDialect dialect, char escapedChar)

```

Comment:
I'd suggest making `dialect` the first parameter of this method because I think it would be more logical.